ads/google/imaVideo.js

```diff
@@ -317,9 +317,22 @@ export function imaVideo(global, data) {
   setStyle(videoPlayer, 'width', '100%');
   setStyle(videoPlayer, 'height', '100%');
   setStyle(videoPlayer, 'background-color', 'black');
-  videoPlayer.setAttribute('src', data.src);
   videoPlayer.setAttribute('poster', data.poster);
   videoPlayer.setAttribute('playsinline', true);
+  // Set video player source, first based on source child elements if present.
+  // If not, then based on data-src.
+  if (data.sources) {
```

Comment:
OK can do - I handled this by making data-src the first source child node, then added the others after that.